### PR TITLE
Changes made in Wiki Edit

### PR DESCRIPTION
**Pull request by arpitar**
DIsplayed line numbers by default in the Wiki Edit section

[#OSF-4467]

### CODE REVIEW

File: website/addons/wiki/static/WikiEditor.js [hunk 1]
```diff
@@ -36,6 +38,9 @@ ko.bindingHandlers.ace = {
     }
 };
 
+
```
Comment (chennan47) [inline website/addons/wiki/static/WikiEditor.js]: Those blank lines are not necessary.

File: website/addons/wiki/static/WikiEditor.js [hunk 2]
```diff
@@ -113,10 +118,11 @@ function ViewModel(url, viewText) {
 
     self.wikisDiffer = function(wiki1, wiki2) {
         // Handle inconsistencies in newline notation
-        var clean1 = typeof wiki1 === 'string' ?
-            wiki1.replace(/(\r\n|\n|\r)/gm, '\n') : '';
-        var clean2 = typeof wiki2 === 'string' ?
-            wiki2.replace(/(\r\n|\n|\r)/gm, '\n') : '';
+        var clean1 = typeof wiki1 === 'string' ? 
+         wiki1.replace(/(\r\n|\n|\r)/gm, '\n'):' ';
```
Comment (chennan47) [inline website/addons/wiki/static/WikiEditor.js]: Check code style guide and fix the indent.

File: website/addons/wiki/templates/edit.mako [hunk 3]
```diff
@@ -152,8 +152,8 @@
                                       </a></li>
                                   <!-- /ko -->
                               </ul>
-                              <div id="wmd-button-bar"></div>
 
+                              <div id="wmd-button-bar" style="display:inline"></div>
```
Comment (chennan47) [inline website/addons/wiki/templates/edit.mako]: Avoid inline style and u can put it inside #wmd-button-bar css class.

File: website/static/css/pages/wiki-page.css [hunk 4]
```diff
@@ -6,6 +7,18 @@
...
+}
+
+.linenum p:before {
+      counter-increment: line;
+      content: counter(line) ' ';
+      display: inline-block;
+
+}
+
+
```
Comment (chennan47) [inline website/static/css/pages/wiki-page.css]: one extra blank line.

File: website/static/css/pages/wiki-page.css [hunk 5]
```diff
@@ -82,6 +95,8 @@
     flex-wrap: wrap;
 }
 
+
+
```
Comment (chennan47) [inline website/static/css/pages/wiki-page.css]: those are not necessary

File: website/addons/wiki/static/WikiEditor.js [hunk 6]
```diff
@@ -113,10 +115,10 @@ function ViewModel(url, viewText) {
...
-            wiki1.replace(/(\r\n|\n|\r)/gm, '\n') : '';
-        var clean2 = typeof wiki2 === 'string' ?
-            wiki2.replace(/(\r\n|\n|\r)/gm, '\n') : '';
+        var clean1 = typeof wiki1 === 'string' ?

+         wiki1.replace(/(\r\n|\n|\r)/gm, '\n'):' ';
+        
var clean2 = typeof wiki2 === 'string' ?

+          wiki2.replace(/(\r\n|\n|\r)/gm, '\n'):' ';
```
Comment (chennan47) [inline website/addons/wiki/static/WikiEditor.js]: I don't think u need a space for this empty string but u will need a space before and after ":" sign, check the osf code guideline.

**Comment by chennan47** [comment]
@arpitar Looks good to me. Please address the minor changes needed. I will move to cr2.

**Comment by arpitar** [comment]
@chennan47 Hi, I did not add the style inside the wmd-button-bar css id as I realized it was something that I was trying to fix the issue of showing line numbers in the edit mode, however I didn't really need to do it as changes had to be made elsewhere. Hence I deleted it altogether. So I should let that be right?

**Comment by arpitar** [comment]
I will check the osf guideline and correct the changes to that code area you highlighted and update it rightaway. Thank you.

File: website/addons/wiki/static/WikiEditor.js [hunk 7]
```diff
@@ -113,10 +115,10 @@ function ViewModel(url, viewText) {
 
     self.wikisDiffer = function(wiki1, wiki2) {
         // Handle inconsistencies in newline notation
-        var clean1 = typeof wiki1 === 'string' ?
-            wiki1.replace(/(\r\n|\n|\r)/gm, '\n') : '';
-        var clean2 = typeof wiki2 === 'string' ?
-            wiki2.replace(/(\r\n|\n|\r)/gm, '\n') : '';
+        var clean1 = typeof wiki1 === 'string' ?
```
Comment (chennan47) [inline website/addons/wiki/static/WikiEditor.js]: indent is 4.

File: website/static/css/pages/wiki-page.css [hunk 8]
```diff
@@ -1,20 +1,25 @@
...
 }
-
 .markdown-it-view {
     padding: 10px;
 }
-
+.linenum {
+  counter-reset: line;
+}
+.linenum p:before {
```
Comment (samanehsan) [inline website/static/css/pages/wiki-page.css]: I don't see these css classes being used anywhere. Can they be removed?

File: website/static/css/pages/wiki-page.css [hunk 9]
```diff
@@ -284,35 +268,38 @@
...
+.wmd-prompt-dialog {
     border: 1px solid #999999;
     background-color: #F5F5F5;
 }
-
 .wiki-single-heading {
     border: 1px solid #ddd;
 }
-
+#linebtn {
```
Comment (samanehsan) [inline website/static/css/pages/wiki-page.css]: Same as above, is this css being used anywhere? If not, it should be taken out.

**Comment by samanehsan** [comment]
Pass complete @arpitar! I just have a couple of comments :octopus:

**Comment by arpitar** [comment]
Hey Saman,

I would look into the css and fix it. Anything else that you think I should be looking into?

File: website/static/css/pages/wiki-page.css (hunk 8 above)
Comment (samanehsan) [inline website/static/css/pages/wiki-page.css]: Can you remove the `linenum` and `linenum p:before` CSS classes if they aren't being used anywhere?

**Comment by samanehsan** [comment]
@arpitar, can you remove the linenum and linenum p:before CSS classes from `wiki-page.css` if they aren't being used anywhere? Also, can you look into the failing test? If it's unrelated to your changes you may need to pull from develop again. Thanks! :sunny:

**Comment by arpitar** [comment]
Hey Saman,

What’s the failing test which you are referring to? Could you please elaborate?
My modification only involves displaying the line numbers in the edit section.

Thank you!

**Comment by samanehsan** [comment]
Thanks for making those changes @arpitar! A test was failing in your previous commit, but it looks like it's passing now so this is good to go. Done with code review 2! :tada:

**Comment by arpitar** [comment]
Yay that sounds great! :)

**Comment by sloria** [comment]
Pass finished. Code looks good. Further changes pending @erinmayhood 's feedback on https://openscience.atlassian.net/browse/OSF-4467 .

**Pull request merged**